x-pack/plugins/observability_solution/apm/server/utils/es_fields_mappings/transaction.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  TRANSACTION_REPRESENTATIVE_COUNT,
+  TRANSACTION_RESULT,
+  TRANSACTION_SAMPLED,
+  TRANSACTION_ID,
+  TRANSACTION_DURATION,
+  TRANSACTION_TYPE,
+  TRANSACTION_NAME,
+  SERVICE_NODE_NAME,
+  SERVICE_NAME,
+  SERVICE_FRAMEWORK_NAME,
+  TRACE_ID,
+  AGENT_NAME,
+  AGENT_VERSION,
+  EVENT_SUCCESS_COUNT,
+  EventOutcome,
+  EVENT_OUTCOME,
+  PROCESSOR_EVENT,
+  PROCESSOR_NAME,
+  DATA_STREAM_NAMESPACE,
+  DATA_STEAM_TYPE,
+  DATA_STREAM_DATASET,
+  SPAN_ID,
+  OBSERVER_HOSTNAME,
+  OBSERVER_TYPE,
+  OBSERVER_VERSION,
+  OBSERVER_VERSION_MAJOR,
+  AT_TIMESTAMP,
+  LABEL_NAME,
+  LABEL_GC,
+  LABEL_TYPE,
+  LABEL_TELEMETRY_AUTO_VERSION,
+  LABEL_LIFECYCLE_STATE,
+} from '@kbn/apm-types';
+import type { AgentName } from '@kbn/elastic-agent-utils';
+import { TIMESTAMP } from '@kbn/rule-data-utils';
```

Comment:
Something is going on with the span/transaction flyout date
<img width="591" alt="Screenshot 2024-10-01 at 13 50 35" src="https://github.com/user-attachments/assets/4a8deb06-04da-49ad-9807-e9d3d691d106">

I've verified it is looking for `timestamp.us` [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/apm/public/components/shared/transaction_action_menu/sections.ts#L72) 

But it looks like we are mapping `timestamp.us` with this import which is actually the `@timestamp`.

Reply:
Thanks for spotting that! It should be the one from the `'@kbn/apm-types'`, I fixed that.